stwcs/updatewcs/__init__.py

```diff
@@ -75,13 +76,19 @@ def updatewcs(input, vacorr=True, tddcorr=True, npolcorr=True, d2imcorr=True,
               If True, attempt to add astrometric solutions from the
               MAST astrometry database.
               Default value is True.
-
     all_wcs : bool
               This parameter only gets used if `use_db=True` to control
               what WCS solutions from the Astrometry database gets appended
               to the file.  If True, all solutions get appended.  If False,
               only solutions based on the IDCTAB from the file's PRIMARY
               header will be appended.
+    remove_duplicates : bool
+              This parameter only gets used if `use_db=True` to remove any
+              duplicate headerlet extensions.   These extension contain WCS
+              solutions that are identical to the WCS found in other
+              extensions of the image.  The duplications typically occur
+              through errors in the code, but will cause the headerlet functions
+              to fail if left in place.
```

Comment:
Also, "The duplications typically occur through errors in the code..." kind of makes me wonder why is the code not fixed in the first place.

Reply:
The original errors that resulted in the astrometry database pushing duplicate WCSs have been fixed.  However, the impact still remains in the database, and users could still find themselves introducing this problem themselves through use of their own scripts.  This alerts the user to the possibility so that hopefully they can avoid creating this problem themselves.

Reply:
This parameter description was revised to move the explanation into the function level docstring instead.  Fixed in 9098e78.